Utility/DynamicJsonParser.cs

```diff
@@ -135,6 +137,38 @@ private static dynamic GetDynamic(PSObject obj, string numericalPrefix = "n", bo
                     propertyName = propertyName.ToLowerInvariant();
                 }
 
+                // Nuspec contains xml metadata as a string (only applicable to VSTS feeds)
+                if (propertyName == "nuspec")
+                {
+                    XmlDocument xmlDoc = new XmlDocument();
+                    string xmlString = (string)psProperty.Value;
+
+                    if (xmlString[0] != '<')
+                    {
+                        var strIndex = xmlString.IndexOf('<');
+                        xmlString = xmlString.Substring(strIndex, xmlString.Length - 1);
+                    }
+
+                    xmlDoc.LoadXml(xmlString);
+                    XmlNodeList metadataNodeList = xmlDoc.GetElementsByTagName("metadata");
+
+                    foreach (XmlNode node in metadataNodeList)
+                    {
+                        if (node.HasChildNodes)
+                        {
+                            // Adding VSTS feed metadata values to the object containing nupkg property information
+                            for (int i = 0; i < node.ChildNodes.Count; i++)
+                            {
+                                string property = node.ChildNodes[i].Name;
+                                string propertyVal = node.ChildNodes[i].InnerText;
+
+                                object actualPropertyVal = ConvertObject(propertyVal);
+                                ((IDictionary<string, object>)actualObj)[property] = actualPropertyVal;
```

Comment:
>((IDictionary<string, object>)actualObj)[property] = actualPropertyVal; [](start = 32, length = 71)

Is this going to overwrite the earlier assigned property values in this loop? Consider assigning these values conditionally when the specified property is not available in the actualObj.

Reply:
Those values won't be overwriting anything because the json format for the VSTS feed only has those particular properties in nuspec.  The only properties for the json file are "commitId",  "previousCommitId", "nextCommitId", "createdDate", "modifiedDate", "nuspec",  and "_links".